xla/service/xla_compile_main.cc

```diff
@@ -68,29 +72,31 @@ StatusOr<std::string> AotCompileCpuExecutable(
   return result;
 }
 
-#if GOOGLE_CUDA
 StatusOr<std::string> AotCompileGpuExecutable(
     std::unique_ptr<HloModule> hlo_module,
     const gpu::GpuTargetConfig& gpu_target_config,
     const AutotuneResults& autotune_results = AutotuneResults()) {
-  gpu::NVPTXCompiler nvptx_compiler;
+#if GOOGLE_CUDA
```

Comment:
On cpu builds, there is a problem because `gpu_compiler` is undefined then.

Reply:
oops, yup. Will fix. Thanks